website/docs/r/container_app.html.markdown

```diff
@@ -361,6 +363,18 @@ A `custom_domain` block supports the following:
 
 ---
 
+A `ip_security_restriction` block supports the following:
+
+* `action` - (Required) The IP-filter action. `Allow` or `Deny`
```

Comment:
Can we add a note here to call out that the rules must all be `Allow` or all be `Deny`, and a mix is not supported by the API at this time?
```suggestion
* `action` - (Required) The IP-filter action. `Allow` or `Deny`.

~> **NOTE:** The `action` types in an all `ip_security_restriction` blocks must be the same for the `ingress`, mixing `Allow` and `Deny` rules is not currently supported by the service.
```